### ISSUE TITLE
[v0.2.x] Roadmap

### ISSUE THREAD

**Issue by kaiakz** [kaiakz/walv#9]
In the version v0.2.x, walv is rewrited from bootstrap/jquery to vuejs/elementui. See the branch: [vue](https://github.com/kaiakz/walv/tree/vue).
It has a beautiful interface and more functions. 
It will support:

- what v0.1.x can do
- C/Python code generation
- Save/Load project(indexedDB)
- Coming soon...

**Comment by kaiakz** [comment]
@embeddedt @kisvegabor @amirgon
Sorry. This project has been delayed for several months. I will continue to develop. The MVP product will release soon.

**Comment by excitedbox** [comment]
Oh man this looks awesome. Can´t wait for image and Canvas support ;)

**Comment by kaiakz** [comment]
preview:
![preview](https://user-images.githubusercontent.com/51747223/75126997-84043300-56f7-11ea-8b54-3a5603cec9b7.gif)

**Comment by kisvegabor** [comment]
It looks awesome!
Do you plan to write a few lines about it on the Forum?

**Comment by kaiakz** [comment]
@kisvegabor 
Of Course! Before that, I need to write a guide and make more implementation.
Do you have any suggestion?

**Comment by kisvegabor** [comment]
> Do you have any suggestion?

Do you have plans for a "style editor"? (Note that, v7 will have a [new style system](https://github.com/littlevgl/lvgl/issues/1243)).

**Comment by kaiakz** [comment]
@kisvegabor 
Yes, I do. 
walv wraps some commonly used functions(I store them in a `table`), called `template`.
For example, to change the `x` of the `btn0` to 88 , walv will use the `template` (`id`.set_x(`integer`)) to generate the `btn0.set_x(88)`, and then send it to lv_micropython. 
Users don't have to care about the code, they just need to click the mouse, walv will generate some commands for lv_micropython by the `template`.

walv can do anythings that lv_micropython supports. No matter v6 or v7, we just need to add the new `template`.

**Comment by kisvegabor** [comment]
I see!

Awesome work, congratulation!

**Comment by excitedbox** [comment]
Love the way this new editor looks.

I noticed that the delete function only works with sub items of a widget but not the actual widget itself.

I think having a drop down for adding widgets is kind of slow to use because it adds extra steps. I think a item pallate like the tools in photoshop are done would be better. Basically a bunch of small icon buttons that you can click to add widgets to the screen. As long as the icons are kept small (32x32 maybe) it wouldn´t take up much space either.

**Comment by kaiakz** [comment]
> I noticed that the delete function only works with sub items of a widget but not the actual widget itself.

I fixed it now. 



> I think having a drop down for adding widgets is kind of slow to use because it adds extra steps. I think a item pallate like the tools in photoshop are done would be better. Basically a bunch of small icon buttons that you can click to add widgets to the screen. As long as the icons are kept small (32x32 maybe) it wouldn´t take up much space either.

That's a good idea(for v0.3.x)! Thanks
